.github/workflows/ci.yml

```diff
@@ -78,7 +80,8 @@ jobs:
           sudo apt-get update && \
           sudo apt-get install -yq libasound2-dev libsoxr-dev libvorbisidec-dev libvorbis-dev libflac-dev \
           libopus-dev alsa-utils libpulse-dev libavahi-client-dev libssl-dev \
-          avahi-daemon debhelper ccache expat cppcheck doxygen graphviz
+          avahi-daemon debhelper ccache expat cppcheck doxygen graphviz \
+          libpipewire-0.3-dev pipewire-pulse
       - name: doxygen
         run: |
```

Comment:
I think these are not needed and/or duplicates `ibdbus-1-dev libjack-dev libavahi-client-dev libatomic1  libvorbis-dev libopus-dev libpulse-dev libflac-dev libsoxr-dev libexpat1-dev`